### ISSUE TITLE
The email is not returning while sign in with Facebook.

### ISSUE THREAD

**Issue by DhavalAlbiorix** [cuongdevjs/reactjs-social-login#102]
**Describe the bug**
I've passed the email in the scope but still didn't get it while sign in with Facebook.

**To Reproduce**
<LoginSocialFacebook
              appId={process.env.REACT_APP_FB_APP_ID || 'key'}
              scope="public_profile, email, user_birthday"
              fieldsProfile={
                'id,first_name,last_name,middle_name,name,name_format,picture,short_name,gender,email'
              }
              onLoginStart={() => {
                console.log("Login STart : ");
              }}
              redirect_uri={REDIRECT_URI}
              onResolve={({ provider, data }: IResolveParams) => {
                console.log("Provider : ", provider);
                console.log("Data : ", data);
              }}
              onReject={err => {
                console.log("Error Facebook : ", err);
              }}
              return_scopes
            >
              <Button
                className="meta-button"
                variant="outlined"
                startIcon={svgIcon("metaIcon")}
                disabled={!isLocationAllowed}
              >
                Sign in with Meta
              </Button>
            </LoginSocialFacebook>

**Expected behavior**
Other than email I'm getting all the fields.

**Screenshots**
![image](https://user-images.githubusercontent.com/87973227/230834500-ccb209a8-6307-41a7-ac0d-ba73b9a87dbd.png)

**Additional context**
Do we need business verification for getting an email while sign in with faceook?.

**Comment by cuongdevjs** [comment]
Have you tried logging in https://react-social-login.netlify.app?

**Comment by DhavalAlbiorix** [comment]
Yes, I'm able to login in that website. But can you please let me know then what I missed in my code?

**Comment by cuongdevjs** [comment]
Sure. It is my code:
````
<LoginSocialFacebook
	appId={""}
	onLoginStart={onLoginStart}
	fieldsProfile={"id,first_name,last_name,middle_name,name,name_format,picture,short_name,email"}
	redirect_uri={REDIRECT_URI}
	onResolve={({ provider, data }) => {}}
	onReject={(err) => { console.log(err); }}
>
	<FacebookLoginButton />
</LoginSocialFacebook>
````

**Comment by cuongdevjs** [comment]
Lets me know if it works

**Comment by DhavalAlbiorix** [comment]
Still not able to get an email, do we need to require business verification for email permission?

**Comment by cuongdevjs** [comment]
I am not sure, maybe you need to configure your Facebook application

**Comment by DhavalAlbiorix** [comment]
I've already configured it but not sure why it's not returning

**Comment by cuongdevjs** [comment]
Have you added email in the permission config?
<img width="1355" alt="Screenshot 2023-04-11 at 11 48 35" src="https://user-images.githubusercontent.com/33604446/231058942-1d9386e3-e0ad-4dcd-bd89-11acdaba5da8.png">

**Comment by DhavalAlbiorix** [comment]
Yes but in the document it says we don't require business verification for email permission but I think you did it that's why it's auto granted.

**Comment by cuongdevjs** [comment]
Oh, yes. For a long time, I don't create any new applications on Facebook, a lot of policies have been updated, but you can try to verify the business email.

**Comment by DhavalAlbiorix** [comment]
Yes let me try that.

**Comment by cuongdevjs** [comment]
Ok, now I will close this issue due to it doesn't relevant to the library. Please comfortable continuing this thread if you have any questions.
